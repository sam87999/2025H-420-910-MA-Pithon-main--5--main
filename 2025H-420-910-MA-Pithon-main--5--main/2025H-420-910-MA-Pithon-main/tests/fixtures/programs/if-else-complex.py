# Test de Maxime Bourret
def check_number(x):
    if x > 0:
        if x % 2 == 0:
            return "positive even"
        else:
            return "positive odd"
    else:
        if x == 0:
            return "zero"
        else:
            return "negative"

print(check_number(4))
print(check_number(7))
print(check_number(0)) 