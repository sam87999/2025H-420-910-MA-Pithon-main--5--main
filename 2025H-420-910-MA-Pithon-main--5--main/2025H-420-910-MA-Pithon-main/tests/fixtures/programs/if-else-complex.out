positive even
positive odd
zero
